.azure-pipelines/windows-visual-studio.yml

```diff
@@ -0,0 +1,66 @@
+steps:
+  - script: |
+      :: Use Windows CRLF line endings for checked-out text files
+      git config --global core.autocrlf true
+      set
+    displayName: Print environment variables
+  - checkout: self
+    fetchDepth: 1
+  - powershell: |
+      $LDC_VERSION=1.14.0
+      $LDC_DIR="ldc2-$($LDC_VERSION)-windows-multilib"
+      $LDC_INSTALLER="$($LDC_DIR).7z"
+      $LDC_URL="https://github.com/ldc-developers/ldc/releases/download/v$($LDC_VERSION)/$($LDC_INSTALLER)"
+      If (-not (Test-Path $LDC_INSTALLER)) {
+          Start-FileDownload $LDC_URL -FileName $LDC_INSTALLER
+      }
+      7z x $LDC_INSTALLER > $null
+      If (-not (Test-Path $LDC_DIR/bin/ldmd2.exe)) {
+          echo "Unexpected LDC installation, $LDC_INSTALLER/bin/ldmd2.exe missing"
+      }
+
+      $VISUALD_VER="v0.45.1-rc2"
+      # Download & install Visual D (needs admin rights?)
+      $VISUALD_INSTALLER="VisualD-$(VISUALD_VER).exe"
+      $VISUALD_URL="https://github.com/dlang/visuald/releases/download/$($VISUALD_VER)/$($VISUALD_INSTALLER)"
+      If (-not (Test-Path $VISUALD_INSTALLER)) {
+          Start-FileDownload $VISUALD_URL -FileName $VISUALD_INSTALLER
+      }
+
+  - script: |
+      echo on
+      call "%VSINSTALLDIR%\VC\Auxiliary\Build\vcvarsall.bat" %ARCH%
+      set LDC_VERSION=1.14.0
+      set LDC_DIR=ldc2-%LDC_VERSION%-windows-multilib
+      set VISUALD_VER=v0.45.1-rc2
+      set VISUALD_INSTALLER=VisualD-%VISUALD_VER%.exe
+      set PWD="%cd%"
+
+      .\%VISUALD_INSTALLER% /S
+      # configure DMD path
+      reg add "HKLM\SOFTWARE\DMD" /v InstallationFolder /t REG_SZ /d %PWD%\dmd2 /reg:32 /f
+      # configure LDC path
+      reg add "HKLM\SOFTWARE\LDC" /v InstallationFolder /t REG_SZ /d %PWD%\%LDC_DIR% /reg:32 /f
+
+      # TODO: disable link dependencies monitoring, fails on AppVeyor server
+      reg add "HKLM\SOFTWARE\Microsoft\VisualStudio\14.0\ToolsOptionsPages\Projects\Visual D Settings" /v optlinkDeps /t REG_DWORD /d 0 /reg:32 /f
+
+      # build via VS projects with LDC
+      cd src
+      # unfortunately, multilib version of LDC and Visual D don't work together seamlessly (anymore?)
+      set LDC_ARGS="/p:ExternalLinkerOptions=phobos2-ldc.lib druntime-ldc.lib"
```

Comment:
The ExternalLinkerOption should not be necessary anymore with later versions of Visual D.